consensus/polybft/validators_snapshot.go

```diff
@@ -179,9 +176,15 @@ func (v *validatorsSnapshotCache) computeSnapshot(
 		return nil, fmt.Errorf("failed to decode extra from the block#%d: %w", header.Number, err)
 	}
 
-	snapshot := existingSnapshot
+	var snapshot AccountSet
+
+	var snapshotEpoch uint64
```

Comment:
Nitpick (spare single line :stuck_out_tongue: )

```suggestion
	var  (
	    snapshot AccountSet
            snapshotEpoch uint64
        )
```

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/997/commits/a4709904d8b5cfc71dab457a4bf18bbb1338367b